pepagent/pepagent.py

```diff
@@ -166,43 +209,45 @@ def get_project(
             _LOGGER.info("Files haven't been downloaded, returning empty project")
             return peppy.Project()
 
-        if found_prj is not None:
+        if found_prj:
             _LOGGER.info(f"Project has been found: {found_prj[0]}")
             project_value = found_prj[1]
             return peppy.Project(project_dict=project_value)
         else:
-            _LOGGER.warn(
+            _LOGGER.warning(
                 f"No project found for supplied input. Did you supply a valid namespace and project? {sql_q}"
             )
-            return None
+            return peppy.Project()
```

Comment:
I like this. It might break some logic, however. How do we check for an empty project in `peppy.Project()`? for example, `pephub` specifically looks for `None` being returned from this function and will return a `404` code in response.

Reply:
A thought: Maybe we can write a method in `peppy` for `__bool__` to return `False` if no samples exist? That way we retain conditional abilities. See here: https://www.pythontutorial.net/python-oop/python-__bool__/

Reply:
I am not sure about peppy. Are there any cases in peppy when we will need empty project? If no, then we can add it

Reply:
I think creating an empty project could be valuable...

Reply:
I agree. I wonder though if it's not very intuitive since we are asking the function to give us a `Project` and if it doesn't exist, it returns one anyways (it just doesn't have any information or samples). So it sort of behaves like you wouldn't expect. I'd almost expect a `NotFound` exception or a warning and `None`.

In pephub I am using the fact that this function returns `None` for nonexistent projects to to raise 404's when they are not in the database. If I could convert an empty project to a Falsy with `bool(peppy.Project())` then I could check that way. 

... Or, I guess I could just use the `check_project_exists()` function...